inst/extdata/french_broad_data.csv

```diff
@@ -0,0 +1,291 @@
+"date.time","DO.obs","DO.deficit","depth","k.O2","light"
+NA,"mg L^-1","mg L^-1","m","m d^-1","J m^-2 s^-1"
+"date.time","DO.obs","DO.deficit","depth","k.O2","light"
+"2012-08-30 00:00:00",7.39,2.67581239853204,1.5,10,0
```

Comment:
I pretend that all dates are UTC (I don't combine data sets from different time zones usually), otherwise you can sometimes accidentally have a time that doesn't exist (it's that missing hour during "spring forward", for example)